### DISCUSSION TITLE
swell.card.validateExpiry not working as expected

### DISCUSSION THREAD

**Discussion by Ramxnchv** [swellstores/community#218] [Help] [answered]
Hello,

I'm using swell.card.validateExpiry method to validate my card and it always returns 'false' regardless of input (i'm using documentation example)

![image](https://user-images.githubusercontent.com/38292919/232493810-393e3a2b-c6d6-4daf-9011-78b98d78745e.png)

![image](https://user-images.githubusercontent.com/38292919/232493929-688cbea1-f667-4400-98ee-0fa5107cc956.png)

Other methods like card number and cvc validators are working fine and return true

**Comment by sags95** [comment]
@Ramxnchv I've created a task for the team to look into this. I'll be able to provide updates as we investigate the issue.

**Comment by ericingram** [comment, marked as answer]
@Ramxnchv The function signature is different than in your example. Try this:

```
swell.cart.validateExpiry(cardExpiry.month, cardExpiry.year)
```

**Comment by Ramxnchv** [comment]
@ericingram Thanks, i think the problem is in src/card.js the validateExpiry function receives 2 arguments but in types/index.d.ts it receives only one string. Anyway since all these validations are called inside the createToken function this is not really important, I just wanted to display errors to the user before submitting the form

![image](https://user-images.githubusercontent.com/38292919/232556996-c93b4b45-542c-4efd-9503-9abcfe017b85.png)

**Reply by ericingram** [reply]
Good catch!
